src/Contexts/AuthContext.tsx

```diff
@@ -113,6 +146,11 @@ export function AuthProvider({ children }: any) {
     signOut(auth)
       .then(() => {
         setCurrentUser(undefined);
+        toast.success("You are logged out", {
+          autoClose: 500,
+          pauseOnHover: true,
+          theme: "dark",
```

Comment:
Can you change the theme to "colored" so it's the same as the other toasts?

Reply:
done